data/json/monsters/zed_resort.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "mon_zombie_resort_dancer",
+    "type": "MONSTER",
+    "name": "sleek zombie",
+    "description": "This zombie is rather sleek and barely clothed.",
+    "copy-from": "mon_zombie",
+    "looks_like": "mon_zombie",
+    "hp": 25,
```

Comment:
Why is this zombie so fragile?

Reply:
I was going to make rather weak zombie. If that's too low, what value would you suggest?

Reply:
Why would they be weaker than any other adult human zombie? If you feel they really must be, I wouldn't go lower than about 70 HP, but I can see no reason why a physically fit and muscular dancer would be more fragile than an office worker.

For reference, a zombie child has 40 hp and an anklebiter, which is a toddler, has 35.

Reply:
Fair enough. I will use standart HP.